oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -334,13 +338,22 @@ public IdToken idTokenWithAudience(String targetAudience, List<IdTokenProvider.O
     return IdToken.create(rawToken);
   }
 
-  private HttpResponse getMetadataResponse(String url) throws IOException {
+  private HttpResponse getMetadataResponse(String url, RequestType requestType) throws IOException {
     GenericUrl genericUrl = new GenericUrl(url);
     HttpRequest request =
         transportFactory.create().createRequestFactory().buildGetRequest(genericUrl);
     JsonObjectParser parser = new JsonObjectParser(OAuth2Utils.JSON_FACTORY);
     request.setParser(parser);
     request.getHeaders().set(METADATA_FLAVOR, GOOGLE);
+    // do not send metric header for getUniverseDomain and getAccount
+    if (requestType != RequestType.UNSPECIFIED) {
```

Comment:
qq, I see that UNSPECIFIED is also being used to gate some logic in `getGoogleCredentialsMetricsHeader`. Are both unspecified parts doing the same thing?

From a glance, it looks like they're doing two different things (one is not adding a value to the header and this one is not sending the metric headers at all). Thoughts on adding a different header like `RequestType.UNTRACKED` for this use case or maybe having a helper function `shouldSendMetricHeader()`?